src/nevermined/utils/JwtUtils.ts

```diff
@@ -34,7 +36,7 @@ export class EthSignJWT extends SignJWT {
         let input = ethers.utils.arrayify(sign)
 
         // TODO: remove once migration to new gateway is complete
-        if (!isEtherSign && process.env.OLD_GATEWAY === 'true') {
+        if (!isEtherSign && config.oldGateway) {
```

Comment:
can we make it true by default if it is not set? that way we keep it internal and create a breaking change